src/services/anchor-service.ts

```diff
@@ -495,10 +509,7 @@ export class AnchorService {
       } for stream ${candidate.streamId.toString()}: ${err}`
       logger.err(msg)
       Metrics.count(METRIC_NAMES.ERROR_IPFS, 1)
-      await this.requestRepository.updateRequests({ status: RS.FAILED, message: msg }, [
```

Comment:
this change is interesting.  I think it's probably fine and makes sense though - even if the anchor doesn't get to IPFS it's data can successfully get to Ceramic via the CAR file so that's fine.

So is the behavior now that the anchor is marked as COMPLETED even if the publish to ipfs/pubsub fails?

Reply:
Yes. As you know, passing witness CAR is part of a bigger attempt to eliminate excessive reliance on IPFS/pubsub. This change makes it explicit. If you can not announce an anchor over pubsub, or can not store a record on IPFS, it should not be a showstopper for the anchoring process.